nixos/modules/services/hardware/auto-epp.nix

```diff
@@ -0,0 +1,59 @@
+{ config, lib, pkgs, ... }:
+with lib;
```

Comment:
There are ongoing efforts to avoid `with lib;` statements. I won't block the PR on this, but would you mind changing it out for an `inherit (lib) ...;` for the most commonly used items, and just prepending `lib.` to everything that's only used once or twice?

Reply:
ok, all addressed except commit splitting, let's see if it's good to go first then i'll split